src/cql/r4/OMTKLogic.cql

```diff
@@ -172,10 +173,13 @@ define function GetConversionFactor(ingredientCode Code, dailyDose Quantity, dos
     )
     when 6813 then ( /*	Methadone */
       case
-        when dailyDose.value between 1 and 20 then 4
-        when dailyDose.value between 21 and 40 then 8
-        when dailyDose.value between 41 and 60 then 10
-        when dailyDose.value >= 61 then 12
+        /*
+         * note Truncate will get integer part of the argument
+         */
+        when Truncate(dailyDose.value) between 1 and 20 then 4
```

Comment:
I don't understand the need to index into the small set of results {4,8,10,12}.  Obviously it didn't previously work with a `dailyDose.value` of say 20.5, but truncating that would now return a 4 vs. 8, which seems significant.